Gordon360/Models/CCT/Context/ICCTContextProcedures.cs

```diff
@@ -69,6 +70,7 @@ public partial interface ICCTContextProcedures
         Task<List<GET_ALL_ROOMS_BY_IDResult>> GET_ALL_ROOMS_BY_IDAsync(string user_id, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
         Task<List<GET_ALL_USERS_BY_ROOM_IDResult>> GET_ALL_USERS_BY_ROOM_IDAsync(int? room_id, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
         Task<List<GET_BIRTH_DATE_BY_IDResult>> GET_BIRTH_DATE_BY_IDAsync(string ID, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
+        Task<List<GET_CURRENT_CHAPELSKIPSResult>> GET_CURRENT_CHAPELSKIPSAsync(string username, OutputParameter<int> returnValue = null, CancellationToken cancellationToken = default);
```

Comment:
Please refresh the efpt config and exclude this SP from the mapping, since it is not in the scope of your project.

Reply:
I have it in my current version branch, I honestly don't know how it got deleted?